hardhat.config.ts

```diff
@@ -95,6 +95,7 @@ const config: HardhatUserConfig = {
         url:
           process.env.POLYGON_URL ||
           "https://polygon-mainnet.g.alchemy.com/v2/4rzRS2MH5LIunV6cejmLhQelv_Vd82rq",
+        blockNumber: 35_400_000,
```

Comment:
We experimented with pinning a block number before. There are benefits to it, but the big problem is that core ToucanProtocol contracts are upgradeable. Pinning the `OffsetHelper` to a block number means the `OffsetHelper` is not tested against the latest version of Toucan. 

Which can (and in the past has) resulted in problems.

Reply:
got it, removed.